pages/dkp/konvoy/2.3/getting-started/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: layout.pug
-navigationTitle: Install
-title: Install
+navigationTitle: getting-started
```

Comment:
I think this is the "title" that will appear in the left-hand navigation bar, so maybe `Getting started` would fit here too.